### PR TITLE
Task/259 v3

### PR DESCRIPTION
**Pull request by t3easy**
* **Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
bug fix

* **What is the current behavior?** (You can also link to an open issue here)
Some task that are added by before/afterTask are not visible if they are not in the same all.

* **What is the new behavior (if this is a feature change)?**
 Show before/after tasks of the same and all apps

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,11 +18,6 @@
         "friendsofphp/php-cs-fixer": "^2.14",
         "sphpdox/sphpdox": "dev-master"
     },
-    "extra": {
```
Comment (simonschaufi) [inline composer.json]: @sabbelasichon just introduced that. What's the purpose?

Comment (t3easy) [reply composer.json]: oh, sorry that was a rebase of my task branch on branch 2.0

Comment (t3easy) [reply composer.json]: I revented that commit so the squash and merge will not change composer.json

**Comment by t3easy** [comment]
> Please remove https://github.com/TYPO3/Surf/blob/master/.editorconfig#L14 as it's not needed any more
Sorry, but I don't get the relation of this patch to that editorconfig setting.

**Comment by simonschaufi** [comment]
I introduced that in this commit https://github.com/TYPO3/Surf/commit/deca74af1676803d64f8e41439cf2657b4db4278 for phpstorm not to trim the whitespaces but since you fixed the generated output, there is no need any more

**Comment by simonschaufi** [comment]
@t3easy now you forgot to change the title

**Comment by t3easy** [comment]
Ok, but I changed the commit message:
https://github.com/TYPO3/Surf/commit/6845d7eea3f5b1b7deee5443be063b0da5766535

**Pull request merged**